library/alloc/benches/vec.rs

```diff
@@ -732,3 +732,18 @@ fn bench_flat_map_collect(b: &mut Bencher) {
     let v = vec![777u32; 500000];
     b.iter(|| v.iter().flat_map(|color| color.rotate_left(8).to_be_bytes()).collect::<Vec<_>>());
 }
+
+#[bench]
+fn bench_retain_100000(b: &mut Bencher) {
+    let v = (1..=100000).collect::<Vec<u32>>();
+    b.iter(|| {
+        let mut v = v.clone();
```

Comment:
calling `.clone()` inside the benchmark loop should be avoided since it can introduce allocator noise. It's better to just refill the vec.

Reply:
Thanks but the PR was merged, should I submit another PR to fix it?

Reply:
I'll take care of it. I just wanted to leave a note.